testing/kuttl/e2e/exporter-password-change/00-assert.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+# Check that all containers in the instance pod are ready
```

Comment:
To double-check my understanding: we have 00--create-cluster.yaml (maybe rename since it applies _and_ asserts?), which asserts that the pod is running; but this script verifies that all the containers are in the right state?

Or put another way: is there a chance the containers won't be / what happens if they aren't?

Reply:
> maybe rename since it applies and asserts?

I'd rather not rename the file - I think the TestStep is to create the cluster and that step has multiple parts 1) create and 2) assert that the expected resources are created.
I do see how something like `create-and-assert` would be more clear, but would we name every TestStep `<do thing>-and-<assert thing>`? Maybe the fact that it is a TestStep means it does a thing and asserts

> Or put another way: is there a chance the containers won't be / what happens if they aren't?

yes, the containers in the pod might not all be ready, so the `ContainersReady` condition wouldn't be found. At this point, the test would fail